staging/src/k8s.io/cri-client/SECURITY_CONTACTS

```diff
@@ -0,0 +1,20 @@
+# Defined below are the security contacts for this repo.
+#
+# They are the contact point for the Product Security Committee to reach out
+# to for triaging and handling of incoming issues.
+#
+# The below names agree to abide by the
+# [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
+# and will be removed and replaced if they violate that agreement.
+#
+# DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
+# INSTRUCTIONS AT https://kubernetes.io/security/
+
+tallclair
+yujuhong
+dims
+cjcullen
+joelsmith
+liggitt
+philips
+tallclair
```

Comment:
tim is here twice =) Maybe delete a second tim

@tallclair

Reply:
Good, fixed :+1: